src/constants/message.js

```diff
@@ -3,6 +3,15 @@ const MESSAGE = {
   REQUIRE_NUMBER_ONLY: '숫자만 입력해주세요',
   CVC_TOOLTIP: '카드 뒷면의 3자리 숫자를 입력해주세요',
   CARD_ADD_COMPLETE: '카드 등록이 완료되었습니다',
+  CARD_ADD_FAILED: '카드를 추가하지 못했어요! 다시 시도해주세요',
```

Comment:
취향일순 있지만 `api.js`처럼 계층화해서 메시지를 관리한다면 어떨까요?

```suggestion
CARD: {
  ADD: {
    COMPLETE: '카드 등록이 완료되었습니다',
    FAILED: '카드를 추가하지 못했어요! 다시 시도해주세요'
  },
  DELETE,
  UPDATE
}
```

Reply:
헉 이런 계층화도 좋은 방법이네요! 제안해주셔서 감사합니다!

Reply:
해당 커밋에서 적용하였습니다!
https://github.com/woowacourse/react-payments/pull/32/commits/e2edf3eb5cf2b30099b0d41a84368f9329054f18